geomstats/geometry/hyperbolic.py

```diff
@@ -476,7 +477,7 @@ def random_uniform(self, n_samples=1, bound=1.):
 
 
 class HyperbolicMetric(RiemannianMetric):
-    """Class for the Hyperbolic metric."""
+    """Class that defines operations using a Riemannian metric."""
```

Comment:
Maybe replace Riemannian with Hyperbolic? The above comment is generic for Riemannian metrics, but I this class provides functions related to that specific metric